tests/tests-core/src/main/java/com/gentics/mesh/test/context/MeshTestContext.java

```diff
@@ -171,6 +169,8 @@ public void setup(MeshTestSetting settings) throws Exception {
 		default:
 			break;
 		}
+		boolean setAdminPassword = settings.optionChanger() != MeshCoreOptionChanger.INITIAL_ADMIN_PASSWORD;
```

Comment:
After each distinct test we clean up. The cleanup includes the indices wipe, but does not restart Mesh. So on the second test we have the running Mesh, and no data. When we start setting the data up, Mesh sees the ES as ready and sends all the created entities to it (this is happening because of the events processing happening on each succeeded Tx, and not manually as before). With no index definition yet existing in ES, ES starts generating the definitions, based on the data itself, which differs from the ones being explicitly sent by the setupIndexHandlers() request, and as a consequence - every second ES-related test fails because of wrong request body JSON.
Yes, this was a tough one to detect.